.pre-commit-config.yaml

```diff
@@ -47,7 +47,7 @@ repos:
 
   # Lint Dockerfiles for errors and to ensure best practices
   - repo: https://github.com/AleksaC/hadolint-py
-    rev: v2.12.0.3
+    rev: v2.12.1b3
     hooks:
       - id: hadolint
```

Comment:
Tried out the official `hadolint` [pre-commit hook](https://github.com/hadolint/hadolint/blob/master/docs/INTEGRATION.md#pre-commit) using this configuration:

```yaml
  - repo: https://github.com/hadolint/hadolint
    rev: v2.12.0
    hooks:
      - id: hadolint-docker
```

And it worked locally with `pre-commit run -a hadolint-docker`. Should we switch over if `hadolint-py` is unmaintained?

Reply:
I don't think `hadolint-docker` will work without Docker installed and running locally will it?

And in any case, it's  the underlying `hadolint` that's been abandoned (I misspoke in the initial comment) so I'm wondering if there are alternatives for Dockerfile linting out there.